.travis.yml

```diff
@@ -100,7 +103,7 @@ script:
   - |
     if [[ $BUILD_DOCS == false ]]; then
       export MPL_REPO_DIR=$PWD  # needed for pep8-conformance test of the examples
-      gdb -return-child-result -batch -ex r -ex bt --args python tests.py -s --processes=$NPROC --process-timeout=300 $TEST_ARGS
+      gdb -return-child-result -batch -ex r -ex bt --args python tests.py $TEST_ARGS
```

Comment:
Yes. At this stage I was just experimenting to see it that is the issue. The code coverage is build against my branch at https://codecov.io/github/jenshnielsen/matplotlib/commits unfortunately I still have issues getting it to work correctly. Either the links to files don't work or they are not there at all.